### PR TITLE
feat: Files & Docs table created

### PR DESCRIPTION
**Pull request by Wennio-Oliveira**
# Description

Created a table for Files & Docs on My recent activities (Dashboard page)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Manually tested.

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have already been accepted and merged

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Frontend](https://nr-spar-webapp-63-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar-webapp/actions/workflows/merge-main.yml)

### CODE REVIEW

File: frontend/src/components/RecentActivities/index.tsx [hunk 1]
```diff
@@ -92,7 +102,19 @@ const RecentActivities = () => {
               </div>
               )}
             </TabPanel>
-            <TabPanel>Placeholder</TabPanel>
+            <TabPanel>
+              <FilesDocsTable
+              elements={filesDocsItems}
+              headers={fileDocstableHeaders}
+              />
+              <div className="empty-recent-activity-files-docs">
```
Comment (mgaseta) [inline frontend/src/components/RecentActivities/index.tsx:110]: Isn't necessary a check here to decide if this should be shown or not? (Just like line 95 of this file)

**Review by mgaseta — CHANGES_REQUESTED** [review]

File: frontend/src/components/FilesDocsTable/styles.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
+a {
+    color: inherit;
+    text-decoration: inherit
```
[the remark this answers is not in the document]
Comment (Wennio-Oliveira) [reply frontend/src/components/FilesDocsTable/styles.css:3]: fixed

File: frontend/src/components/RecentActivities/index.tsx [hunk 3]
```diff
@@ -92,7 +102,19 @@ const RecentActivities = () => {
               </div>
               )}
             </TabPanel>
-            <TabPanel>Placeholder</TabPanel>
+            <TabPanel>
+              <FilesDocsTable
+              elements={filesDocsItems}
+              headers={fileDocstableHeaders}
```
[the remark this answers is not in the document]
Comment (Wennio-Oliveira) [reply frontend/src/components/RecentActivities/index.tsx:108]: fixed

File: frontend/src/components/RecentActivities/index.tsx (hunk 1 above)
Comment (Wennio-Oliveira) [reply frontend/src/components/RecentActivities/index.tsx:110]: fixed

**Review by mgaseta — APPROVED** [review]
LGTM! ✔🥇

**Review by lucasbuenoencora — APPROVED** [review]
LGTM! :1st_place_medal:

**Pull request merged**